include/pybind11/cast.h

```diff
@@ -1224,7 +1241,7 @@ class unpacking_collector {
         int _[] = { 0, (process(args_list, std::forward<Ts>(values)), 0)... };
         ignore_unused(_);
 
-        m_args = object(PyList_AsTuple(args_list.ptr()), false);
+        m_args = std::move(args_list);
```

Comment:
The way the conversion was done in the original `PYBIND11_OBJECT_CVT` macro, the move constructor was more efficient, so the `std::move` here resulted in smaller binary size. With the reworked `CVT` macro, the lvalue ref constructor is equally good, so the move isn't strictly needed any more, but it doesn't hurt either (`args_list`'s lifetime ends here anyway).